VotingApplication/VotingApplication.Web/Views/Create/ConfirmStep.cshtml

```diff
@@ -74,39 +73,43 @@
                 <md-list-item class="md-3-line">
 
                     <md-icon ng-if="expiryIsValid()" md-font-icon="fa-check" class="fa md-primary fa-2x" alt="check"></md-icon>
-                    <md-icon ng-if="!expiryIsValid()" md-font-icon="fa-exclamation-triangle" class="fa md-warn fa-2x" alt="exclamation triangle"></md-icon>
+                    <md-icon ng-if="!expiryIsValid()" md-font-icon="fa-exclamation-triangle" class="fa vo-warn-text fa-2x" alt="exclamation triangle"></md-icon>
 
                     <div class="md-list-item-text">
 
-                        <div ng-if=" newpoll.expiry">
-                            <h3>{{newPoll.Expiry}}</h3>
+                        <div ng-if="newPoll.DoesExpire">
+                            <h3>{{formatUtcPollExpiry()}}</h3>
```

Comment:
This can be done with a filter, keeping the display logic in the html. e.g. `newpoll.Expiry | momentDisplayFilter`.